examples/ts/stablecoin/initiate-burn-order.ts

```diff
@@ -10,20 +10,26 @@ require('dotenv').config({ path: '../../../.env' });
 
 /**
  * Step 1. GET /assets API
- * Step 2. GET /constants API
- * Step 3. POST /order API
- * Step 4. sendMany
+ *     Retrieves:
+ *       - The list of assets required for burning
```

Comment:
```suggestion
 *       - The list of supported assets for initiating burning
```

Reply:
Did not mention "supported assets" because we are fetching usd asset as well